web/src/components/Layout.tsx

```diff
@@ -113,13 +116,16 @@ type PropsType = {
  */
 const Layout = ({ asideStickyTop = 0, footer, header, toolbar, modal, children }: PropsType): JSX.Element => {
   const modalVisible = !!modal
+  const hideSidebar = window.location.href.includes(POIS_ROUTE)
```

Comment:
ok we already have a separate ticket for adding toolbar into the bottomActionSheet

Reply:
I don't understand this comment :see_no_evil: Or rather, what one has to do with another.

Reply:
I mean maybe we want to have at least the feedback bar somewhere on the pois page thats why we have a separate ticket. Just removing it and not replacing it could be not what we want in the end?

Reply:
but you did remove it before my comment already? just in a complicated way, right?

Reply:
indeed